src/commands/distribute/release.ts

```diff
@@ -14,13 +14,16 @@ import { AppCenterClient, models, clientRequest, ClientResponse } from "../../ut
 import { out } from "../../util/interaction";
 import { inspect } from "util";
 import * as _ from "lodash";
-import * as Request from "request";
 import * as Path from "path";
 import * as Pfs from "../../util/misc/promisfied-fs";
-import { DefaultApp } from "../../util/profile";
+import { DefaultApp, getUser } from "../../util/profile";
+import { getPortalUploadLink, getPortalPatchUploadLink } from "../../util/portal/portal-helper";
 import { getDistributionGroup, addGroupToRelease } from "./lib/distribute-util";
-import * as fs from "fs";
-import * as stream from "stream";
+import { McFusUploader, McFile } from "./lib/mc-fus-uploader/mc-fus-uploader";
+import { McFusMessageLevel, McFusUploadState } from "./lib/mc-fus-uploader/mc-fus-uploader-types";
+import "abort-controller/polyfill";
```

Comment:
Do not need this anymore, just put import of this lib in uploader

Reply:
Fixed in v-ruurme/fix-mc-fus-feedback